pose_estimator_params:
  keypoints_3D_topic: "/keypoints_3D"
  pose_topic: "/estimated_pose"
  cad_frame_file_name: "$(env HOME)/CadFrames/box.csv"
  noise_bound: 40
  cbar2: 1
  estimate_scaling: false
  rotation_max_iterations: 100
  rotatation_gnc_factor: 1.4
  rotation_cost_threshold: 0.005